src/main/java/com/coalesce/ttb/blocks/TextLoader.java

```diff
@@ -4,40 +4,45 @@
 import com.google.common.util.concurrent.ListenableFuture;
 import com.google.common.util.concurrent.SettableFuture;
 import org.bukkit.Location;
-import org.bukkit.Material;
 import org.bukkit.util.Vector;
 
 import java.awt.*;
 import java.awt.image.BufferedImage;
 import java.util.HashSet;
 import java.util.Set;
 
-public class TextLoader {
-
-	private TextToBlock plugin;
-	private Location origin;
-	private TextLoader.TextOrientation orientation = TextOrientation.NORTH;
-	private String text;
-	private Material material = Material.STONE;
-	private String fontName;
-	private float fontSize = 12;
+public final class TextLoader {
+	
+	private TextDirection direction;
```

Comment:
Would prefer if this was named "TextOrientation" as direction is 2D, and orientation is 3D